package.json

```diff
@@ -30,17 +30,17 @@
   "homepage": "https://github.com/jedireza/mongo-models",
   "devDependencies": {
     "body-parser": "1.x.x",
-    "code": "5.x.x",
+    "@hapi/code": "5.3.x",
```

Comment:
You can leave this at `5.x.x` since it will bring in the latest minor version during installation.

Reply:
will revert that.